AssignAll/AssignAll/AssignAllMembers/RegionsToAnalyzeProvider.cs

```diff
@@ -10,8 +10,8 @@ namespace AssignAll.AssignAllMembers
 {
     internal static class RegionsToAnalyzeProvider
     {
-        private static readonly Regex AssignAllDisableRegex = new Regex(@"^// AssignAll disable$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
-        private static readonly Regex AssignAllEnableRegex = new Regex(@"^// AssignAll enable$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex AssignAllDisableRegex = new Regex(@"^// ?AssignAll disable$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex AssignAllEnableRegex = new Regex(@"^// ?AssignAll enable$", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
I would maybe just expand to `\s*` to allow any whitespace or no whitespace, both leading and trailing. In particular trailing whitespace would be easy to miss.

I'm not 100% sure, but I believe these regexes are only applied to single lines at a time, so `\s` will not accidentally match newline characters.

If so, we can change to:
```
^//\s*AssignAll disable\s*$
^//\s*AssignAll enable\s*$
```

If `SingleLineCommentTrivia` somehow allow newlines, then we can tighten it a bit more:

```
^//[^\S\r\n]*AssignAll disable[^\S\r\n]*$
^//[^\S\r\n]*AssignAll enable[^\S\r\n]*$
```
https://stackoverflow.com/a/3469155/134761

Reply:
Sure, extended the regex. I think we can assume that if there are line breaks in-between, it will not be considered as a single-line comment. I've tried to throw in some `\n` and `\r` in the test and it was just a compile error since it breaks a comment.

Reply:
Great